server/server.js

```diff
@@ -1,14 +1,25 @@
 const express = require('express');
 const path = require('path');
-const MySQL = require('mysql');
+const mysql = require('mysql');
+const db = require('../db/connection.js');
 
+const statsQuery = `USE funding_stats; SELECT pledged, goal, currency, backers, 
+                    deadline, category, city, state, country FROM campaigns WHERE id = ?;`;
 const port = 3000;
 const app = express();
 
 app.use(express.static(path.resolve(__dirname, '../../funding-widget/public')));
 app.use(express.json());
 app.use(express.urlencoded({ extended: true }));
 
+app.get('/campaigns/:campaignId/stats', (req, res) => { // handle requests for a given campaign's stats
+  let campaign = parseInt(req.params.campaignId); // parse out the campaign's unique id
+
+  db.query(statsQuery, [campaign], (error, results) => { // retrieve campaign stats from the database
```

Comment:
I second Sarah's comment about putting some error handling in here (I wouldn't have noticed it on my own).

Reply:
I have added an if() block to log any errors that come through on the database query. Thanks @pickles27 and @jordanstholmes for catching that.